src/components/composer/composer/composer.ts

```diff
@@ -400,6 +400,38 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.processTokenAtCursor();
   }
 
+  onDblClick() {
+    if (this.env.model.getters.isReadonly()) {
+      return;
+    }
+    const composerContent = this.env.model.getters.getCurrentContent();
+    const isValidFormula =
+      composerContent.startsWith("=") && this.env.model.getters.getCurrentTokens().length > 0;
```

Comment:
there is something I don't understand here: if the content starts with "=" then getCurrentTokens.length will always be >0 since we have the '=' token. the only other way to have tokens empty would be (to my knowledge) to have the tokenize crash but if it crashed then this bit of code should not work either. So I don't realy understand what it does... @hokolomopo  I think you introduced this condition with the multiline, do you have a clue ?

Reply:
I don't remember at all this line, and the git blame points on you on some composer fix https://github.com/odoo/o-spreadsheet/pull/1771/files 😛 

But I'd say that you're right and that the second condition is useless ? I don't see how we could have a content and no tokens...
![image](https://github.com/odoo/o-spreadsheet/assets/31045223/d46efd2a-3be1-4f35-8768-9bd7c1df1d9f)

Reply:
damn it, I haven't checked properly, sorry :/
Anyway, I think we can get rid of that condition...

Reply:
I deleted them 🙂